lib/main.js

```diff
@@ -301,8 +344,8 @@ function signText(text) {
   }
 
   let certName = selectCert(userCerts, text);
```

Comment:
I think the current name is OK, so I will keep it, but don't have objections if you change it after merge.